packages/resolver/lib/sources/truffle/Deployed.ts

```diff
@@ -1,6 +1,6 @@
 const web3Utils = require("web3-utils");
 import semver from "semver";
-import RangeUtils from "@truffle/compile-solidity/compilerSupplier/rangeUtils";
+import * as RangeUtils from "@truffle/compile-solidity/dist/compilerSupplier/rangeUtils";
```

Comment:
...isn't there some way to make this work without `/dist/`?  Is that coming in one of the later PRs?

Reply:
😢 

(Resolving this to move discussion to [other comment thread](https://github.com/trufflesuite/truffle/pull/4223#discussion_r688190965))